src/platform/intel/ace/lib/dma.c

```diff
@@ -56,7 +56,7 @@ SHARED_DATA struct dma dma[] = {
 		.caps		= DMA_CAP_HDA,
 		.devs		= DMA_DEV_HOST,
 		.channels	= DT_PROP(DT_NODELABEL(hda_host_in), dma_channels),
-		.period_count	= HDA_DMA_BUFFER_PERIOD_COUNT,
+		.period_count	= 2 * HDA_DMA_BUFFER_PERIOD_COUNT,
```

Comment:
A couple of points to commit message:

1. Please use "platform: ace: update buffer period count"
This helps reviewers to immediately see this is a change specific to one platform.

2. The reason for change is still a bit lacking. Double compared to what? Why not just increase HDA_DMA_BUFFER_PERIOD_COUNT byt two? These maybe obvious, but please explain in git commit message.